packages/hix/lib/Hix/Optparse.hs

```diff
@@ -34,6 +36,9 @@ import Hix.Managed.Cabal.Data.HackageRepo (HackageIndexState, HackageName)
 import Hix.Managed.Data.BuildConfig (SpecialBuildHandlers (..))
 import Hix.Managed.Data.SpecialMaintHandlers (SpecialMaintHandlers (..))
 
+pathUserOption :: ReadM PathUser
+pathUserOption = PathUser . Text.pack <$> str
```

Comment:
```suggestion
pathUserOption = str
```

since `str` has type `IsString s => ReadM s`, this should work implicitly. Probably fine to just use `str` where you'd use `pathUserOption`.

Reply:
oooh indeed! pretty cool